src/cmd/wallet.rs

```diff
@@ -231,6 +232,7 @@ pub struct SpendOpt {
 impl SpendOpt {
     pub fn spend_coins<D: BatchDatabase, Cs: CoinSelectionAlgorithm<D>, Ctx: TxBuilderContext>(
         self,
+        wallet_dir: &std::path::Path,
```

Comment:
spend_coins now takes `wallet_dir` in order to load the config, and then save a PSBT onto SD card